server/settings/publish_plugins.py

```diff
@@ -1200,7 +1225,7 @@ class PublishPuginsModel(BaseSettingsModel):
         "use_hardlinks": False
     },
     "CleanUp": {
-        "paterns": [],
+        "paterns": [],  # codespell:ignore paterns
```

Comment:
```suggestion
        "paterns": [],  # codespell:ignore patterns
```

Reply:
I had to add `# codespell:ignore paterns` so that codespell don't error on this line. 
I couldn't fix the typo as it's not related to this PR.